__tests__/users.test.ts

```diff
@@ -38,7 +38,7 @@ afterAll(async () => {
 
 describe("[GET] /api/users/?filter", () => {
```

Comment:
Can we add some tests here for getting only unapproved users as well as only approved users? Just to make sure that the query parameters are working correctly